info/buildSrc/src/main/kotlin/org/cqfn/diktat/generation/docs/GenerationDocs.kt

```diff
@@ -131,6 +133,27 @@ enum class FindType {
     ;
 }
 
+private fun handleHyperlinks(line: String): String {
+    var correctedString = line
+    if (correctedString.contains(HYPERLINKS)) {
+        val hyperlinkSubString = HYPERLINKS.find(correctedString)!!.groupValues
+        hyperlinkSubString.forEach {
+            var link = Regex("""\(http.*\)""").find(it)!!.groupValues.first().drop(1).dropLast(1) // drop ( and )
```

Comment:
Isn't it better to introduce captured groups in `HYPERLINKS` regex and retrieve them there instead of having two additional regexes?